constellations/gemini-admin/src/location/location.sass

```diff
@@ -42,6 +43,8 @@
 
 #view-location #location-row
     border-bottom: 1px solid $colorGrayDark
+    display: flex
+    align-items: center
```

Comment:
Let's give each row more padding on the top and bottom to space things out a bit more.

CSS ordering should be display, align, padding then border-bottom

Reply:
The `<a>` Edit button should have some margin before it and the Room column.